cilium-1.15.advisories.yaml

```diff
@@ -156,6 +156,10 @@ advisories:
             componentType: go-module
             componentLocation: /usr/bin/hubble-relay
             scanner: grype
+      - timestamp: 2024-05-04T16:36:53Z
+        type: true-positive-determination
```

Comment:
This might be the single largest advisory event we've ever had 😄 . Happy to remove it if the size is off-putting to anyone! But otherwise... it's all true. 😇

Reply:
My main concern is how this will look in an UI.

Reply:
Perhaps we could just keep the information that is relevant for users.

Reply:
Okay, sounds good — I pushed a shortened version